extension/src/openvic-extension/singletons/MapItemSingleton.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <godot_cpp/classes/object.hpp>
+#include <godot_cpp/variant/packed_int32_array.hpp>
```

Comment:
I don't think this is needed anymore, maybe replace it with `packed_byte_array`?